e2e/scenario/scenario.go

```diff
@@ -44,39 +44,50 @@ func GetScenariosForSuite(ctx context.Context, suiteConfig *suite.Config) (Table
 // its return value to the slice returned by this function.
 func (t *Template) scenarios() []*Scenario {
 	return []*Scenario{
+		// block for ubuntu 1804
 		t.ubuntu1804(),
+		t.ubuntu1804gpu(),
+		t.ubuntu1804_azurecni(),
+		t.ubuntu1804gpu_azurecni(),
+		t.ubuntu1804ChronyRestarts(),
+
+		// block for ubuntu 2204
 		t.ubuntu2204(),
-		t.marinerv2(),
-		t.azurelinuxv2(),
 		t.ubuntu2204ARM64(),
-		t.marinerv2ARM64(),
-		t.azurelinuxv2ARM64(),
-		t.ubuntu1804gpu(),
-		t.marinerv2gpu(),
-		t.azurelinuxv2gpu(),
 		t.ubuntu2204CustomSysctls(),
-		t.marinerv2CustomSysctls(),
-		t.azurelinuxv2CustomSysctls(),
 		t.ubuntu2204Wasm(),
-		t.marinerv2Wasm(),
-		t.azurelinuxv2Wasm(),
-		t.ubuntu1804_azurecni(),
-		t.marinerv2_azurecni(),
-		t.azurelinuxv2_azurecni(),
-		t.ubuntu1804gpu_azurecni(),
-		t.marinerv2gpu_azurecni(),
-		t.azurelinuxv2gpu_azurecni(),
 		t.ubuntu2204gpuNoDriver(),
 		t.ubuntu2204CustomCATrust(),
 		t.ubuntu2204ArtifactStreaming(),
 		t.ubuntu2204privatekubepkg(),
 		t.ubuntu2204AirGap(),
 		t.ubuntu2204ContainerdURL(),
 		t.ubuntu2204ContainerdVersion(),
-		t.azurelinuxv2ARM64AirGap(),
+		t.ubuntu2204ChronyRestarts(),
+
+		// block for mariner v2
+		t.marinerv2(),
+		t.marinerv2ARM64(),
+		t.marinerv2gpu(),
+		t.marinerv2CustomSysctls(),
+		t.marinerv2Wasm(),
+		t.marinerv2_azurecni(),
+		t.marinerv2gpu_azurecni(),
 		t.marinerv2AirGap(),
-		t.azurelinuxv2AirGap(),
 		t.marinerv2ARM64AirGap(),
+		t.marinerv2ChronyRestarts(),
+
+		// block for azurelinux v2
+		t.azurelinuxv2(),
+		t.azurelinuxv2ARM64(),
+		t.azurelinuxv2gpu(),
+		t.azurelinuxv2CustomSysctls(),
+		t.azurelinuxv2Wasm(),
+		t.azurelinuxv2_azurecni(),
+		t.azurelinuxv2gpu_azurecni(),
+		t.azurelinuxv2ARM64AirGap(),
+		t.azurelinuxv2AirGap(),
+		t.azurelinuxv2ChronyRestarts(),
```

Comment:
thanks a lot @timmy-wright  for this clean up  It makes it 110% more readable and we will easily be able to spot where test are missing and what is being tested where!